TestObjectiveDropbox/TestObjectiveDropbox_iOSTests/TeamRoutesTests.m

```diff
@@ -14,28 +15,45 @@ @implementation TeamRoutesTests {
     DBTeamClient* _teamClient;
 }
 
+- (NSString *)environmentVariableForKey:(NSString *)key {
```

Comment:
This seems a duplicated method as the one defined in FileRoutesTests.m. Might be cleaner to make a common until method to be called in both files.